portfolio/src/main/webapp/index.html

```diff
@@ -226,6 +229,32 @@ <h2>Connect with me</h2>
   </section>
 </div>
 
+<!-- Modal for setting nickname -->
+<div class="modal fade" id="nickname-modal" tabindex="-1" role="dialog"
+    aria-labelledby="nickname-modal-title" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="nickname-modal-title">Set Nickname</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <form action="/login" method="POST" id="nickname-form">
```

Comment:
nit: I think we want to use lowercase for all html attributes, elements, etc.

See https://google.github.io/styleguide/htmlcssguide.html#Capitalization

method="post"

Reply:
Thanks for the reference.
Changes made.